docs/tutorial/packages.rst

```diff
@@ -51,12 +51,21 @@ external ``C`` libraries, the following options are available:
   before the Idris modules, for example to support linking with a
   ``C`` library.
 
-+ ``libs = <libs>``, which gives a comma separated list of libraries
++ ``libs = <libs>``, which takes a comma separated list of libraries
   which must be present for the package to be usable.
 
-+ ``objs = <objs>``, which gives a comma separated list of additional
++ ``objs = <objs>``, which takes a comma separated list of additional
   object files to be installed, perhaps generated by the ``Makefile``.
 
+For testing Idris packages, there is a rudimentary testing harness built into Idris.
+This allows for testing functions that are written in context of ``IO`` to be run automatically in a fresh environment on your computer.
+The field is:
+
++ ``tests = <test functions>``, which takes the qualified names of all tests to be run.
+
+.. IMPORTANT::
+  The tests must also be added to the list of modules in ``modules``.
```

Comment:
What does this mean? A test is a term, not a module. Do you mean that the modules containing the tests must be added?